interfaces.go

```diff
@@ -238,12 +239,15 @@ func (m *Mechanism) SetUnhandled() {
 
 // Exception specifies an error that occurred.
 type Exception struct {
-	Type       string      `json:"type,omitempty"`  // used as the main issue title
-	Value      string      `json:"value,omitempty"` // used as the main issue subtitle
-	Module     string      `json:"module,omitempty"`
-	ThreadID   uint64      `json:"thread_id,omitempty"`
-	Stacktrace *Stacktrace `json:"stacktrace,omitempty"`
-	Mechanism  *Mechanism  `json:"mechanism,omitempty"`
+	Type             string      `json:"type,omitempty"`  // used as the main issue title
+	Value            string      `json:"value,omitempty"` // used as the main issue subtitle
+	Module           string      `json:"module,omitempty"`
+	ThreadID         uint64      `json:"thread_id,omitempty"`
+	IsExceptionGroup bool        `json:"is_exception_group,omitempty"`
+	ExceptionID      int         `json:"exception_id"`
+	ParentID         *int        `json:"parent_id,omitempty"`
```

Comment:
According to the RFC, these fields need to go on the [`Mechansim`](https://github.com/getsentry/sentry-go/blob/master/interfaces.go#L223-L230).